README.md

```diff
@@ -192,6 +192,34 @@ def get_special_floors_list(default_list):
   return default_list
 ```
 
+## Monsters
+`DowntimeDungeon_monsters` - gvar_id
+Configures the list of potential monsters
+
+The corresponding gvar if set should contain a function called `get_monster_list` which takes the argument `default_list` and returns a `list` to use on the server.
+
+The configure_monster() function takes the list of arguments shown below.
+
+```py
+using(
+    monster_tools = "XXX"

```

Comment:
urhhh but i meant the actual gvar id 😅

Reply:
```suggestion
    monster_tools = "d62a1569-c48b-4e80-8702-96d05ffaacb9"

```

Reply:
oh, i see lmao

Reply:
Btw, I think its importing monsters, not monster_tools
The file I copied it from just calls it monster_tools because monsters is an existing variable